test/osutils.jl

```diff
@@ -13,9 +13,9 @@
 @test Base.is_unix(:FreeBSD)
 @test_throws ArgumentError Base.is_unix(:BeOS)
 if !is_windows()
-    @test Sys.windows_version() === (0, 0)
+    @test Sys.windows_version() == v"0.0.0"
 else
-    @test (Sys.windows_version()::Tuple{Int,Int})[1] > 0
+    @test Sys.windows_version() > v"0.0.0"
```

Comment:
Test isn't quite the same as before. Previously the test would check if the major revision was > 0. Maybe it would be better to do:
```julia
@test Sys.windows_version() >= v"1.0.0-"
```